### PR TITLE
Update phpdoc completions based on the phpDocumentor latest docs at http://phpdoc.org/docs/latest/index.html

### PR DESCRIPTION
**Pull request by gerardroche**
Closes https://github.com/spadgos/sublime-jsdocs/pull/311

    /**
     * @api
     * @author [name] <[<email address>]>
     * @copyright [description]
     * @deprecated [<version>] [<description>]
     * @example [URI] [<description>]
     * @filesource
     * @global [type] [name | description]
     * @ignore [<description>]
     * @internal [<description>]
     * @license [<url>] [name]
     * @link [URI] [<description>]
     * @method [return type] [name]([[type] [parameter]<, ...>]) [<description>]
     * @package [level 1]\[level 2]\[etc.]
     * @param [type] $name [<description>]
     * @property [type] $name [<description>]
     * @property-read [type] $name [<description>]
     * @property-write [type] $name [<description>]
     * @return [type] [<description>]
     * @see [URI | FQSEN] [<description>]
     * @since [version> [<description>]
     * @source [start-line [<number-of-lines>]] [<description>]
     * @throws [type] [<description>]
     * @todo [description]
     * @uses [FQSEN] [<description>]
     * @var [type] [<description>]
     * @version [<vector>] [<description>]
     *
     * @category [description]
     * @subpackage [name]
     */

### CODE REVIEW

File: php.sublime-completions [hunk 1]
```diff
@@ -1,34 +1,62 @@
...
+    {"trigger":"@property-read",    "contents":"@property-read ${1:[type]} \\$${2:name} ${3:[<description>]}"},
+    {"trigger":"@property-write",   "contents":"@property-write ${1:[type]} \\$${2:name} ${3:[<description>]}"},
+    {"trigger":"@return",           "contents":"@return ${1:[type]} ${2:[<description>]}"},
+    {"trigger":"@see",              "contents":"@see ${1:[URI | FQSEN]} ${2:[<description>]}"},
+    {"trigger":"@since",            "contents":"@since ${1:[version>} ${2:[<description>]}"},
+    {"trigger":"@source",           "contents":"@source ${1:[start-line [<number-of-lines>]]} ${2:[<description>]}"},
+    {"trigger":"@throws",           "contents":"@throws ${1:[type]} ${2:[<description>]}"},
+    {"trigger":"@todo",             "contents":"@todo ${1:[description]}"},
+    {"trigger":"@uses",             "contents":"@uses ${1:[FQSEN]} ${2:[<description>]}"},
+    {"trigger":"var",               "contents":"@var ${1:[type]} ${2:[<description>]}"},
```
Comment (spadgos) [inline php.sublime-completions]: Is this meant to be `@var` ?

**Comment by gerardroche** [comment]
@spadgos Good catch. Fixed.

File: php.sublime-completions (hunk 1 above)
Comment (gerardroche) [inline php.sublime-completions]: @spadgos Good catch. Fixed.

File: php.sublime-completions [hunk 2]
```diff
@@ -1,34 +1,62 @@
...
+  "completions": [
+
+    // Based on the phpDocumentor latest documentation at http://phpdoc.org/docs/latest/index.html
+
+    // Not able to add trigger descriptions, e.g. "trigger" : "@api\tPHPDoc",
+    // because there are sublime text bugs with handing the @ symbol when used
+    // with a description.
+
+    {"trigger":"@api",              "contents":"@api"},
+    {"trigger":"@author",           "contents":"@author ${1:[name]} ${2/.+/</}${2:[<email address>]}${2/.+/>/}"},
```
Comment (spadgos) [inline php.sublime-completions]: clever!

**Comment by spadgos** [comment]
I've rebased and merged this as 2a0addb

I added a couple of changes here 551b8c0. Basically, just making sure that each of the placeholders has `[ .. ]` around them (so that reparsing the docblock will pick them up), and removing `< >` from the inside of placeholders, since we already have the square brackets.

**Pull request closed without merging**